src/scripts/icons/generateJson.ts

```diff
@@ -17,7 +17,7 @@ try {
 
   const manifest = generateManifest();
   // Print manifest to stdout so that scripts can consume it
-  console.log(JSON.stringify(manifest, undefined, 2));
```

Comment:
@PKief What does "undefined" mean?

Reply:
@okineadev it's a replacer function: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#replacer. Undefined means, that nothing is replaced.

Reply:
Okau